definitions/npm/redux-rsaa-middleware_v1.x.x/flow_v0.66.x-/redux-rsaa-middleware_v1.x.x.js

```diff
@@ -0,0 +1,29 @@
+declare module 'redux-rsaa-middleware' {
+  import type { Middleware } from 'redux';
+
+  declare export type RSAAAction = {
+    type: string,
+    payload: RSAAPayload,
+  };
+
+  declare export type RSAAPayload = {
+    method?: string,
+    path?: string,
+    query?: string,
+    error?: Object,
+    errorTime?: string,
+    response?: Object,
+    failureAction?: RSAAAction,
+    successAction?: RSAAAction,
+    startedSendingAction?: RSAAAction,
+    finishedSendingAction?: RSAAAction,
+  };
+
+  declare export type RSAAState = Object;
+  
+  declare export type DispatchAPI<A> = (action: A) => A;
+  
+  declare export type Dispatch<A: { type: $Subtype<string> }> = DispatchAPI<A>;  
+  
+  declare export var RSAAMiddleware : Middleware<RSAAState, RSAAAction, Dispatch<RSAAAction>>;
```

Comment:
Change to `module.exports`

Reply:
I'm curious, why was this necessary?

Reply:
Well it wasn't, that was my sleepy state talking. I just changes it back in a commit. -.-